web3/contract.py

```diff
@@ -958,6 +961,7 @@ class ImplicitContract(ConciseContract):
 
     > contract.functions.withdraw(amount).transact({})
     '''
+    @deprecated_for("Please use the verbose contract syntax. Ex: contract.functions.withdraw(amount).transact({})")
```

Comment:
The deprecation warning will probably read a little weird here, something like "this method was deprecated for: Please use the verbose...".

Maybe the `str` should just start "verbose contract syntax".